.github/workflows/apps_wallet_prod_build.yml

```diff
@@ -39,6 +39,48 @@ jobs:
           echo "artifact_name=${artifact_name}" >> $GITHUB_ENV
       - name: Build Wallet
         run: pnpm wallet build
+      - name: Get Previous Tag
+        id: prev_tag
+        run: |
+          tags=$(git tag --list 'wallet-v*.*.*' --sort=-creatordate)
+          current_tag=$(echo "$tags" | sed -n 1p)
+          prev_tag=$(echo "$tags" | sed -n 2p)
+          if [ -z "$prev_tag" ]; then
+            echo "No previous tag found. Skipping changelog generation."
+            echo "PREV_TAG=none" >> $GITHUB_ENV
+          else
+            echo "PREV_TAG=$prev_tag" >> $GITHUB_ENV
+          fi
+          echo "CURRENT_TAG=$current_tag" >> $GITHUB_ENV
+
+      - name: Generate Changelog
+        id: generate_changelog
+        run: |
+          if [ "${{ env.PREV_TAG }}" = "none" ]; then
+            echo "No previous tag found. Skipping changelog generation."
+            echo "changelog=No previous tag found. Changelog generation skipped." >> $GITHUB_ENV
+          else
+            git log ${{ env.PREV_TAG }}..${{ env.CURRENT_TAG }} --pretty=format:"- %s by @%an in #%h" -- ./apps/wallet > CHANGELOG.md
+            changelog=$(cat CHANGELOG.md)
+            echo "changelog=$changelog" >> $GITHUB_ENV
+          fi
+
+      - name: Get version from tag
+        id: version
+        run: echo "::set-output name=version::${GITHUB_REF#refs/tags/wallet-v}"
+
+      - name: Create GitHub Release
+        uses: elgohr/create-release@v5
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ env.CURRENT_TAG }}
+          release_name: IOTA Wallet v${{ steps.version.outputs.version }}
+          body: |
+            ## Changelog
+            {{ env.changelog }}
+          draft: false
+          prerelease: false
```

Comment:
Are we sure we don't want to have a human review it?

Reply:
The release is merely a formality, it's not related to the actual publishing of the extension or anything like that, you got a fair point though.

@begonaalvarezd wdyt?

Reply:
lets go for `draft: true`, I prefer revieweing the releases myself

Reply:
we decided to make it `draft` always 🫡